build/flaws.js

```diff
@@ -528,7 +528,10 @@ async function fixFixableFlaws(doc, options, document) {
         newSrc = path.basename(destination);
       } catch (error) {
         const { response } = error;
-        if (response && response.statusCode === 404) {
+        if (
+          (response && response.statusCode === 404) ||
+          error.code === "ETIMEDOUT"
```

Comment:
What if the server is just having a bad day? Or your computer might think it's connected to the internet, because it's connected to the router, but it's actually not succeeding in any packets. What would happen in either case is that you missing the opportunity to try to download (and correct) this image. What the correct thing to do in that case, is to try a little bit later. 

However, I think it's quite *plausible* that you're even in this mess because, actually, the domain is gone so it times out (ie. there's not even a server who wants to respond that the image is 404ing). 
And given that there's little harm (you just can't turn the external image into a local image today), perhaps it's just fine. 

The only thing I object to is the "lie" that we're checking for `ETIMEDOUT` but say "Skipping ${flaw.src} (404)". 
So, can you break up this if statement so it becomes something like this:
```javascript
if (statuscode===404) {
  console.log(chalk.yellow(`Skipping ${flaw.src} (404)`));
  continue;
} else if (ETIMEDOUT) {
  console.log(chalk.yellow(`Skipping ${flaw.src} (request timed out)`));
  continue;
} else {
  throw
}
```

Reply:
Or imagine your connection goes away during download …

Reply:
Broke up the warnings